js/mobile-menu.js

```diff
@@ -15,4 +16,28 @@ function hideMobileMenu() {
 
 hamburger.addEventListener('click', displayMobileMenu);
 xButton.addEventListener('click', hideMobileMenu);
-mobileNav.addEventListener('click', hideMobileMenu);
\ No newline at end of file
+mobileNav.addEventListener('click', hideMobileMenu);
+
+// modal for project details
+const projectsDetails = document.querySelectorAll('.see-project');
+const modal = document.querySelector('.modal-project');
+const xModalButton = document.querySelector('.x-cross-modal');
+const detailsButtons = document.querySelectorAll('.modal-buttons-p');
+
+function displayProjectDetails() {
+  modal.style.display = 'block';
+}
+
+function closeProjectDetails() {
+  modal.style.display = 'none';
+}
+
+projectsDetails.forEach((project) => {
+  project.addEventListener('click', displayProjectDetails);
+});
+
+xModalButton.addEventListener('click', closeProjectDetails);
+
+detailsButtons.forEach((button) => {
+  button.addEventListener('click', closeProjectDetails);
+});
```

Comment:
According to the project requirements, you should rebuild the project section using javascript. This implies that the modals also will be loaded dynamically from javascript. in order to make this possible, you would need to implement an array of objects containing all your project information. See the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20use,implement%20the%20popup%20window.)

_NOTE_: All hand-coded projects in your HTML should be removed and only an empty container should be left. 



Furthermore, the design of the pop-up window on the desktop doesn't match that of the Figma design. I recommend you implement it correctly in order to stick with the provided template.

YOUR DESIGN

![modal](https://user-images.githubusercontent.com/84930355/223662171-9326c5b3-b775-44ed-95e9-c0334e7e248d.PNG)


FIGMA DESIGN: [figma](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m8_popup/modal_desktop.png)

Reply:
added desktop-version modal project details

Reply:
adjust the ui similar with figma design